slybot/slybot/spider.py

```diff
@@ -197,14 +192,15 @@ def parse(self, response):
         content_type = response.headers.get('Content-Type', '')
         if isinstance(response, HtmlResponse):
             return self.handle_html(response)
-        elif "application/rss+xml" in content_type:
-            return self.handle_rss(response)
-        else:
-            self.logger.debug(
-                "Ignoring page with content-type=%r: %s" % (content_type,
-                                                            response.url)
-            )
-            return []
+        if (isinstance(response, XmlResponse) or
+                response.url.endswith(('xml', 'xml.gz'))):
```

Comment:
A bit unsafe, a lot of html pages about xml will end in xml. Can we use `.endswith(('.xml', '.xml.gz'))` (notice the extra dots) so it's less likely?